subhub/tests/unit/stripe/customer/test_stripe_customer.py

```diff
@@ -124,46 +124,19 @@ def test_stripe_hub_customer_source_expiring(mocker):
 
 
 def test_stripe_hub_customer_subscription_created(mocker):
-    customer_response = mock(
-        {
-            "id": "cus_00000000000000",
-            "object": "customer",
-            "account_balance": 0,
-            "created": 1563287210,
-            "currency": "usd",
-            "metadata": {"userid": "user123"},
-        },
-        spec=stripe.Customer,
-    )
-    when(stripe.Customer).retrieve(id="cus_00000000000000").thenReturn(
-        customer_response
-    )
     data = {
-        "uid": "user123",
-        "active": True,
-        "subscriptionId": "sub_00000000000000",
-        "subscription_id": "sub_00000000000000",
-        "productName": "subhub",
-        "eventId": "evt_00000000000000",
-        "eventCreatedAt": 1326853478,
-        "messageCreatedAt": int(time.time()),
-        "invoice_id": "in_test123",
-        "plan_amount": 500,
-        "customer_id": "cus_00000000000000",
-        "nickname": "subhub",
-        "created": 1519363457,
-        "canceled_at": 1519680008,
-        "cancel_at": None,
-        "event_type": "customer.subscription.created",
         "event_id": "evt_00000000000000",
-        "cancel_at_period_end": False,
+        "event_type": "customer.created",
+        "email": "user123@tester.com",
+        "customer_id": "cus_00000000000000",
+        "name": "Jon Tester",
+        "user_id": "user123",
     }
-    logger.info("created payload", data=data)
     basket_url = CFG.SALESFORCE_BASKET_URI + CFG.BASKET_API_KEY
     response = mock({"status_code": 200, "text": "Ok"}, spec=requests.Response)
-    when(boto3).client("sns", region_name=CFG.AWS_REGION).thenReturn(MockSnsClient)
+    when(boto3).client("sqs", region_name=CFG.AWS_REGION).thenReturn(MockSqsClient)
```

Comment:
Weird that this changed from Sns to Sqs; maybe explain that to us at some point.

Reply:
good catch, changed.